mapviz_plugins/ui/pointcloud2_config.ui

```diff
@@ -102,8 +111,11 @@
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="bufferSize">
+     <property name="minimum">
+      <number>1</number>
```

Comment:
Isn't a minimum buffer size used for an infinite buffer?

Reply:
That was the intent for odometry, gps, tf, etc. data, but I'm not sure if an infinite buffer makes much sense for point cloud data.  It doesn't take too long for point cloud data to accumulate to the point it becomes impractical to render.

Reply:
In practice, the pointcloud2 plugin is actually really slow.  Having a buffer of more than 10 already has serious performance issues for large point clouds; allowing an infinite buffer was a good way to make Mapviz quickly peg an entire CPU core and then grind to a halt.